macros/tests/basic.rs

```diff
@@ -87,14 +104,19 @@ testcases![
         1:1-1:2
     )
 }",
+// FIXME(eddyb) get replace quotes with backticks and pretify the `expected` list.
+// FIXME(eddyb) the partial result should print as an `S` node.
+    S("aax") => r#"1:3: error: expected ["d", "d", "a", "b", "a", "c"]; partial result:
+1:1-1:3"#;
```

Comment:
And possibly include the string the span refers to so I don't have to think so much...